blackbox/docs/general/builtins/aggregation.rst

```diff
@@ -411,21 +411,21 @@ corresponding to the percentile of the specified fraction::
     +----------------------------+-------------+
     | percentile(position, 0.95) | kind        |
     +----------------------------+-------------+
-    |         5.699999999999999  | Galaxy      |
-    |         4.8                | Planet      |
-    |         3.8499999999999996 | Star System |
+    |                        6.0 | Galaxy      |
```

Comment:
why is this rounded up? Is that normal?

Reply:
See https://github.com/elastic/elasticsearch/pull/28305#issuecomment-362542140